src/target_detector/TargetDetector.cpp

```diff
@@ -40,7 +40,7 @@ gaze point falls into. It then publishes this information with publisher topic "
 #include <geometry_msgs/Vector3.h> 
 #include <limits>
 
-#define GAZE_ERROR 50
+#define GAZE_ERROR -20
```

Comment:
two things here:

 1. what is GAZE_ERROR ?
 2. while using `define`s, please stick to a single convention (either all uppercase or all lowercase, never both)